plugins/block-sharable-procedures/src/events_procedure_parameter_base.ts

```diff
@@ -79,8 +60,8 @@ export abstract class ProcedureParameterBase extends ProcedureBase {
 }
 
 export interface ProcedureParameterPair {
-  procedure: Blockly.procedures.IProcedureModel;
-  parameter: Blockly.procedures.IParameterModel;
+  procedure?: Blockly.procedures.IProcedureModel;
+  parameter?: Blockly.procedures.IParameterModel;
```

Comment:
Kind of funny for this to be called `…Pair`, when it could be just one or indeed none!

Reply:
Haha I don't have a better idea so I think we're sticking with this.

Although when I was ideating "ProcedureClump" did pop into my head, and I find this very amusing :P